### PR TITLE
feat(VETS-CPC-1110): make veterinarian and home page accessible.

### PR DESCRIPTION
**Pull request by kyrusho**
…eVeterinarian's page. No designing or data fetching has been done, only the access to the page. Two files have been modified, router.tsx and path.routes.ts with my comments on each modified line, and one file has been created, the Vet's layout page.

JIRA: link to jira ticket
## Context:
What is the ticket about and why are we doing this change.
## Changes
What are the various changes and what other modules do those changes effect.
This can be bullet point or sentence format.
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
If this is a new UI feature, include screenshots to show reviewers what it looks like. 
##### (The next part is an addition from @nic5694 to help with the issues that some people have been running into in the past couple of days)
Added workspace configurations to VS Code because some people were missing plugins needed to see linting errors they might introduce. This update includes settings for format on save—when you modify a file and save it, VS Code will automatically format the code for you.

**Important**: If you already have a .vscode folder in the root of the petclinic-frontend folder, remove it before pulling from main, or you’ll get a conflict. It’s best that everyone uses this configuration to ensure smoother development and avoid build failures in CI.

**Extension Recommendations**:
Added recommended extensions that you’ll find at the bottom of the Extensions Marketplace in VS Code. Please **install the following**:
        Prettier: For consistent code formatting.
        ESLint: To catch linting errors.
        "try-material-icon-theme": To make it easier to navigate the folder structure with icons.

These tools will help you spot issues early and streamline the development process.
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

File: petclinic-frontend/src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { NavBar } from '@/layouts/AppNavBar.tsx';
+
+// Haitham: created the home page layout
```
Comment (cgerard321) [inline petclinic-frontend/src/pages/Home/Home.tsx:3]: Comment code like this should not be left in PRs.

File: petclinic-frontend/src/pages/Vet/Vet.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { NavBar } from '@/layouts/AppNavBar.tsx';
+
+// Haitham: created the vet page layout
```
Comment (cgerard321) [inline petclinic-frontend/src/pages/Vet/Vet.tsx:3]: Comment code like this should not be left in PRs.

File: petclinic-frontend/src/shared/models/path.routes.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export enum AppRoutePaths {
   Default = '/',
   Inventories = '/inventories',
+  Vet = '/vet', //added vet's route path
```
Comment (cgerard321) [inline petclinic-frontend/src/shared/models/path.routes.ts:4]: Comment code like this should not be left in PRs.

**Review by cgerard321 — COMMENTED** [review]
Update the PR title to match the PR naming convention. 
Fill in the PR template.

File: petclinic-frontend/src/shared/models/path.routes.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 export enum AppRoutePaths {
   Default = '/',
   Inventories = '/inventories',
+  Vet = '/vet',
```
Comment (nic5694) [inline petclinic-frontend/src/shared/models/path.routes.ts:4]: For next time it is always best to append new items at the bottom of a list. If you add it in the middle all the lines are shifted down and the risk of a merge conflict is much greater.

**Review by nic5694 — COMMENTED** [review]
Please see the changes i mentioned in the code before merging with main.

File: petclinic-frontend/src/shared/models/path.routes.ts (hunk 4 above)
Comment (kyrusho) [reply petclinic-frontend/src/shared/models/path.routes.ts:4]: Ok thanks for the tip.

File: petclinic-frontend/src/shared/models/path.routes.ts (hunk 3 above)
Comment (kyrusho) [reply petclinic-frontend/src/shared/models/path.routes.ts]: The comments have been removed.

File: petclinic-frontend/src/pages/Vet/Vet.tsx (hunk 2 above)
Comment (kyrusho) [reply petclinic-frontend/src/pages/Vet/Vet.tsx]: The comments have been removed.

File: petclinic-frontend/src/pages/Home/Home.tsx (hunk 1 above)
Comment (kyrusho) [reply petclinic-frontend/src/pages/Home/Home.tsx]: The comments have been removed.

**Review by nic5694 — APPROVED** [review]
thanks for applying the changes

**Pull request merged**